### DISCUSSION TITLE
Github Student pack validity

### DISCUSSION THREAD

**Discussion by vins-anity** [community/community#144470] [GitHub Education]
### Hi everyone,

I got accepted last october 2, 2023 is it still valid? i couldnt re apply again because it says 

" Hi, Waisnu! You were last verified as a student on Oct 2, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits. "

help

**Comment by davevad93** [comment]
Hi @Waisnu ,

Yes, GitHub Education benefits last for 2 years, which means you'll have access to them until October 2, 2025. After that time, if you want to renew them, you'll have to re-verify your student status.

**Reply by vins-anity** [reply]
The problem here is that I tested my benefits, like jetbrains my license seems expired on that, other benefits like One month for example couldn't detect that I'm under the Student pack, and another example is gitlens specifically gitkraken app, It now proccess me to apply the free trial when I could use it before. I think my benefits are not working well.

**Reply by vins-anity** [reply]
Tho my GH Copilot still auto completes

**Reply by davevad93** [reply]
Hey @Waisnu, that's because `JetBrains` subscription lasts 1 year and has to be renewed annually.

About `GitKraken` that's odd because the license should last as long as you're a student, try going to the [pack offers](https://education.github.com/pack) and check if you can redeem your offers there.
